lib/std/Progress.zig

```diff
@@ -307,7 +307,8 @@ pub const Node = struct {
             @atomicStore(u8, &storage.name[name_len], 0, .monotonic);
 
         const parent_ptr = parentByIndex(free_index);
-        assert(parent_ptr.* == .unused);
+        const current_parent = @atomicLoad(Node.Parent, parent_ptr, .acquire);
```

Comment:
In non-safe builds this introduces an unnecessary acquire-load. Not sure what to do about that.